### ISSUE TITLE
All display sometimes flickered while in cloud

### ISSUE THREAD

**Issue by Kimbyeoungjang** [flybywiresim/a32nx#5670]
### Aircraft Version

Development

### Build info

```json
autopilot-custom-fpm branch

82fc325
```

### Describe the bug

Route. departure from EDDF, destination is EGLL

SOBR6N SOBRA Y180 BITBU Y181 DEMUL M624 BUB L608 LOGAN LOGA1Z

I was fly through cloud and screen sometimes flickered. I thought that is msfs light problem but it is not. other lights all okay but only screen's brightness is problem

### Expected behavior

screen should not flickered

### Steps to reproduce

takeoff and fly through black cloud

### Additional info (optional)

I will add recorded video

**Comment by AdamNZ** [comment]
I've also seen this - with the last few builds. It happens to me when passing through cloud layers, with all glass-fronted instruments affected. Could it be related to the glass reflections code that was added a short while ago?

**Comment by tracernz** [comment]
> 
> 
> I've also seen this - with the last few builds. It happens to me when passing through cloud layers, with all glass-fronted instruments affected. Could it be related to the glass reflections code that was added a short while ago?

No. It appeared with sim update 5 for all emmisives.

**Comment by Venorcis** [comment]
Is https://github.com/flybywiresim/a32nx/pull/5661 in the autopilot-custom-fpm branch build you used?

**Comment by Kimbyeoungjang** [comment]
> Is #5661 in the autopilot-custom-fpm branch build you used?

I believe yes because it is behind 1 commit on master

**Comment by sidnov** [comment]
Also happens on the battery voltage panel on top. It's an issue with shaders that were modified with Sim Update 5

**Comment by tush0701** [comment]
I also noticed the same, and the brightness increases when passing through cloud. check screenshot with and without cloud

With cloud:
![Screenshot (103)](https://user-images.githubusercontent.com/75409941/130102111-6190a40d-0239-4b8a-9342-c6f158b5b184.png)
Check without cloud:
![Screenshot (104)](https://user-images.githubusercontent.com/75409941/130102193-573732dc-0a40-4700-aa56-5a6edf78a1a6.png)
I did not change the brightness, as you can compare both screenshots.

**Comment by sidnov** [comment]
> Probably has something to do with volumetric lighting, that was re-introduced with the hotfix?

Precisely. The shaders were tweaked which is causing excess bloom

**Comment by sidnov** [comment]
Here's a shot flying through a storm with dense clouds, Bloom off in MSFS settings
![image](https://user-images.githubusercontent.com/29005005/130123858-35846334-7119-4fc1-8a0d-cf64fc1c5ec6.png)

**Comment by 2hwk** [comment]
Seemingly fixed by WU6, closing.

**Comment by jbud** [comment]
Possible regression since unknown. I just found this topic after finally searching for an answer

So, it's been bothering me, and I've tried turning off bloom in sim, but nothing works, my PFD gets washed out whenever I go through a cloud. 
version: b4c2044
but been around for months through many updates.
![20231226023923_1](https://github.com/flybywiresim/aircraft/assets/6253799/b212a2a1-3a39-494c-abff-c9e21578c6dd)
![20231226023629_1](https://github.com/flybywiresim/aircraft/assets/6253799/dd716c27-e79c-438f-83bc-c1099cebb38a)
